addons/miscFixes/patchCUP/config.cpp

```diff
@@ -56,6 +56,68 @@ class CfgVehicles {
             };
         };
     };
+    // Tweaks to the GTK Boxer's handling (accel/braking) + HMG swap to M3M + countermeasures move to gunner
+    class Wheeled_APC_F: Car_F {
+        class NewTurret;
+        class Turrets {
+            class MainTurret: NewTurret {
+                class Turrets {};
+            };
+        };
+    };
+    class CUP_Boxer_Base: Wheeled_APC_F {
+        class AnimationSources;
+    };
+    class CUP_Boxer_Base_HMG: CUP_Boxer_Base {
+        class CargoTurret;
+        class Turrets {
+            class MainTurret: NewTurret {
+                weapons[] = {"CUP_Vhmg_M3P_veh","SmokeLauncher"}; // was CUP_Vhmg_M2_veh
+                magazines[] = {"CUP_250Rnd_TE1_Red_Tracer_127x99_M","CUP_250Rnd_TE1_Red_Tracer_127x99_M","CUP_250Rnd_TE1_Red_Tracer_127x99_M","CUP_250Rnd_TE1_Red_Tracer_127x99_M","CUP_250Rnd_TE1_Red_Tracer_127x99_M","SmokeLauncherMag"}; // was CUP_200Rnd_TE1_Red_Tracer_127x99_M
+            };
+            class CommanderTurret: NewTurret {
+                weapons[] = {}; // was "SmokeLauncher"
+                magazines[] = {}; // was "SmokeLauncherMag"
+            };
+        };
+        accelAidForceCoef = 2.3; // was 1.4
+        accelAidForceSpd = 25; // was 5
```

Comment:
```suggestion
        accelAidForceCoef = 2.0; // was 1.4
        accelAidForceSpd = 15; // was 5
```
tone down accl changes?

Reply:
I can test it next year / maybe this weekend depending on some stuff goes, but if you think it doesn't feel out of place than you can merge it

Reply:
Looks good